packages/trader/src/Modules/Trading/Containers/contract-type.tsx

```diff
@@ -41,7 +41,7 @@ const Contract = observer(() => {
             <ContractTypeWidget
                 list={list}
                 name='contract_type'
-                onChange={onChange}
+                onChange={e => onChange({ target: { name: e.target?.name || '', value: e.target?.value } })}
```

Comment:
this seems unnecessary. anyway, we will use package 4 instead of package 3, so i think we can close this PR